build.sbt

```diff
@@ -367,6 +367,10 @@ lazy val protocol = projectMatrix
     settings = jvmDimSettings
   )
   .settings(
+    Compile / packageSrc / mappings := (Compile / packageSrc / mappings).value.collect {
```

Comment:
2 things in a single comment : 

* can you run `scalafmtAll` (format checks are currently not set for *sbt files) 
* this collect could be written as a `filter` (or `filterNot`)